data/queenstack.json)

```diff
@@ -0,0 +1,22 @@
+
+ { "name": "Agada Queen",
+  "type": "personal",
+  "bio": "Data scientist **Business Analyst**. Technical writer"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/maquinbeaury",
+       "icon": "FaTwitter"
+       }
+       {
+      "name": "Follow me on Linkedin",
+      "url": "https://twitter.com/maquinbeaury",
+      "icon": "FaLinkedin"
+      }
+      {
+      "name": "Follow me on Github",
+      "url": "https://twitter.com/maquinbeaury",
```

Comment:
Here also instead of Github profile link, you have given twitter link, please check it ,

Thank you